operator-metadata/metadata/annotations.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   com.redhat.openshift.versions: v4.6
   operators.operatorframework.io.bundle.channel.default.v1: stable
-  operators.operatorframework.io.bundle.channels.v1: stable,amq-streams-1.x,amq-streams-1.8.x
+  operators.operatorframework.io.bundle.channels.v1: stable,amq-streams-1.x,amq-streams-2.0.x
```

Comment:
So, should this be now in `amq-streams-2.x`?

Reply:
Dang! Nice catch!